docs/advanced/index.md

```diff
@@ -0,0 +1,110 @@
+---
+layout: index
+title: Advanced
+navigation_weight: 1
+---
+
+# Caching CDN Requests or Non-200 Responses
+
+If an attempt is made to cache an asset that is on a different origin or
+the asset returns a non-200 response, these modules will throw an error. The
+reason for this default behavior is to ensure that only good and local
+responses are cached.
+
+Depending on how you are using these libraries will depend on how you can add
+support for external resources.
+
+## sw-lib
+
+In sw-lib, you can set up some additional options with a route that will allow
+support for additional status codes and / or based on specific headers.
+
+```javascript
+const cdnCacheStrategy = goog.swlib.staleWhileRevalidate({
+  cacheableResponse: {
+    statuses: [0],
+  },
+});
+goog.swlib.router.registerRoute(new RegExp('^https://cdn.mysite.com/styles/'), cdnCacheStrategy);
+```
+
+This same approach can be used with all of th caching strategies supported by
+sw-lib including `cacheFirst()`, `cacheOnly()`, `networkFirst()`,
+`networkOnly()` and `staleWhileRevalidate()`.
+
+## Lower Level Modules
+
+To cache a response with a status code other than 2XX status code when using
+the lower level modules use the `sw-cacheable-response` plugin with a
```

Comment:
I also have trouble parsing this sentence. This would be a little easier if the qualification were the opening clause. "When using lower level modules, use the `s-c-r` plugin with a `RW` to cache" etc..